FiveCalls/FiveCalls/EditLocationViewController.swift

```diff
@@ -37,6 +38,7 @@ class EditLocationViewController : UIViewController, CLLocationManagerDelegate {
         
         if case .address? = UserLocation.current.locationType {
```

Comment:
I've updated the code to call addressTextFieldChanged, instead of setting isEnabled directly. Also, addressTextFieldChanged makes sure there is something other than white space in the field before enabling the button.